src/explore-education-statistics-frontend/src/components/DownloadCsvButtonWithAnalytics.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import DownloadCsvButton, {
+  DownloadCsvButtonProps,
+} from '@common/modules/table-tool/components/DownloadCsvButton';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+
+const DownloadCsvButtonWithAnalytics = (props: DownloadCsvButtonProps) => {
```

Comment:
No need for new '...WithAnalytics' components, as we only need to use them in one or two places. Just add the `onClick` handler to the required pages.